ast/policy.go

```diff
@@ -1491,6 +1491,21 @@ func (q *Every) Compare(other *Every) int {
 	return q.Body.Compare(other.Body)
 }
 
+// Vars returns the key and val arguments of an every expression,
+// if they are non-nil and not wildcards.
+func (q *Every) Vars() VarSet {
+	r := NewVarSet()
+	if v := q.Value.Value.(Var); !v.IsWildcard() {
+		r.Add(v)
+	}
+	if q.Key != nil {
+		if v := q.Key.Value.(Var); !v.IsWildcard() {
+			r.Add(v)
+		}
+	}
+	return r
```

Comment:
nit: should we use the VarVisitor here? What if someone writes `every k, [x, "foo"] in ...`

Reply:
My thinking was that we'd not allow that, at least not right away. So in the first iteration, key and value need to be plain vars. 

Later on, we could relax that. WDYT?

Reply:
That makes sense, I'm fine with deferring the decision to support compound terms in the key-value position. That said, I could still see value in using the VarVisitor here assuming the check to prohibit compound terms is done during parsing. This way, if/when that limitaiton is lifted, this function will just do the right thing. It's completely your call though--I don't have a strong opinion.

Reply:
Switched to VarVisitor. ✔️